modules/standard/SysError.chpl

```diff
@@ -52,6 +52,164 @@ class SystemError : Error {
   proc writeThis(f) {
     try! f.write(msg);
   }
+
+  proc type fromSyserr(err: syserr, msg: string = "") {
```

Comment:
I don't think the 'msg' argument here should be called 'msg' since it's an elaboration on the error message from strerror. Maybe "details" or something? Obviously it will work no matter what we name this argument. It's just a documentation thing.